examples/features/gracefulstop/server/main.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
Why is this file in this PR?

Reply:
Removed. My fork was not in sync with upstream and tool created the PR